src/NovelRT/Audio/AudioService.cpp

```diff
@@ -332,24 +260,79 @@ std::string AudioService::getALError() {
   }
 }
 
+//for touhou project
```

Comment:
Is comment still necessary?

Reply:
Please remove this comment. This implementation is legacy and is getting nuked anyway, so I don't really need to know why were doing things the way we are in here.